tests/parametric/test_otel_api_interoperability.py

```diff
@@ -227,35 +227,24 @@ def test_span_links_add(self, test_agent, test_library):
             - Test that links can be added with the Datadog API on a span created with the OTel API
         """
         with test_library:
-            with test_library.otel_start_span("otel.span") as otel_span:
-                current_span = test_library.current_span()
+            with test_library.start_span("dd_root") as dd_span:
+                pass
```

Comment:
is this code useful?

Reply:
Yes, this span ("dd_root") is linked to the otel span below. In this test I removed the `http_headers` argument from the Links object. Instead I use on the parent_id to generate a span link. This better aligns with the Otel Start Span API.